RFCs/0029-Passing-TestRunParameters-via-CLI.md

```diff
@@ -0,0 +1,50 @@
+# Passing test run parameters via cli
+
+## Summary
+Test run parameters given via cli are passed to runsettings.
+
+## Motivation
+Every time editing or creating one test run parameter in run settings at a time is a tedious task. So passing test run parameters via cli 
+removes burden of editing run settings every time.
+
+## Syntax
+
+`vstest.console.exe  abc.dll -- TestRunParaeters.Parameter(name="YourParameterName",value="YourParameterValue ")`
```

Comment:
Nit: TestRunParameters*

Also, you might want to include BackSlash('\') I think that was part of the PR changes right ?

Reply:
you mean this right?
TestRunParameters.Parameter(name=\"YourParameterName\",value=\"YourParameterValue\")`